packages/venia-ui/lib/components/Routes/routes.js

```diff
@@ -0,0 +1,30 @@
+import React, { Suspense, lazy } from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import MagentoRoute from '../MagentoRoute';
+
+const CreateAccountPage = lazy(() => import('../CreateAccountPage'));
+const Search = lazy(() => import('../../RootComponents/Search'));
+
+const LOADING_MESSAGE = 'Just a moment...';
+const fallback = <h1>{LOADING_MESSAGE}</h1>;
```

Comment:
Can we use the [LoadingIndicator](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/LoadingIndicator/indicator.js#L9) here? I might be wrong.

Reply:
Fixed in [7a7ef14](https://github.com/magento/pwa-studio/pull/1966/commits/7a7ef14063db684a9103e63de7e826a78e08f1c3).